assets/js/upload.js

```diff
@@ -143,7 +144,9 @@ function setupImageUpload() {
 
   // Fetch on "enter" in url field
   remoteUrl.addEventListener('keydown', event => {
-    if (event.keyCode === 13) {
+    const key = normalizedKeyboardKey(event);
```

Comment:
unnecessary variable declaration, you could simply have written `if (normalizedKeyboardKey(event) === keys.Enter)`